tests/integration/test_groupnameupdate.py

```diff
@@ -8,51 +8,29 @@
 class GroupNameUpdate(BaseIntegrationTest):
 
     @parameterized.expand([
-        # TEST_NAME                    EXEC_USER           TARGET_GRP       RESP
-        ("foreign_as_admin"          , "admin"           , "group_x_id"   , 204),
-        ("own_as_group_admin"        , "group_x_admin"   , "group_x_id"   , 403),
-        ("foreign_as_group_admin"    , "group_x_admin"   , "group_y_id"   , 403),
-        ("invalid_group"             , "admin"           , "duckburgh"    , 403),
-        ("own_as_regular_user"       , "user_a"          , "group_x_id"   , 403),
-        ("foreign_as_regular_user"   , "user_a"          , "group_y_id"   , 403)
+        # TEST_NAME                    EXEC_USER           TARGET_GRP       NEW_GROUPNAME    RESP
+        ("foreign_as_admin"            , "admin"           , "group_x_id"   , "fancy_group"  , 204),
+        ("own_as_group_admin"          , "group_x_admin"   , "group_x_id"   , "fancy_group"  , 403),
+        ("foreign_as_group_admin"      , "group_x_admin"   , "group_y_id"   , "fancy_group"  , 403),
+        ("invalid_group"               , "admin"           , "duckburgh"    , "fancy_group"  , 403),
+        ("own_as_regular_user"         , "user_a"          , "group_x_id"   , "fancy_group"  , 403),
+        ("foreign_as_regular_user"     , "user_a"          , "group_y_id"   , "fancy_group"  , 403),
+        # this throws an IntegrityError instead of 400
+        # ("admin_use_existing_groupname", "admin"           , "group_x_id"   , "group_y"      , 400),
```

Comment:
We should be able to enable this test and merge now, since @MoritzHahn1337 PR was merged, right?
```suggestion
        ("admin_use_existing_groupname", "admin"           , "group_x_id"   , "group_y"      , 400),
```